File: Android/app/src/main/java/com/example/mdp_android/ui/bluetooth/BluetoothViewModel.java
package com.example.mdp_android.ui.bluetooth;

import androidx.lifecycle.LiveData;
import androidx.lifecycle.MutableLiveData;
import androidx.lifecycle.ViewModel;
public class BluetoothViewModel extends ViewModel{
    private MutableLiveData<String> device;

    public BluetoothViewModel() {
        device = new MutableLiveData<>();
        device.setValue("Bluetooth: Not Connected");
    }

    public LiveData<String> getDevice() {
        return device;
    }

    public void setDevice(String device) {
        this.device.setValue(device);
    }

}

File: Android/app/src/main/java/com/example/mdp_android/ui/bluetooth/BluetoothViewModel.kt
package com.example.mdp_android.ui.bluetooth

import androidx.lifecycle.LiveData
import androidx.lifecycle.MutableLiveData
import androidx.lifecycle.ViewModel

class BluetoothViewModel : ViewModel() {

    private val _text = MutableLiveData<String>().apply {
        value = "This is bluetooth Fragment"
    }
    val text: LiveData<String> = _text
}